operators/apicast-community-operator/0.8.0/manifests/apicast-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,305 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "apps.3scale.net/v1alpha1",
+          "kind": "APIcast",
+          "metadata": {
+            "name": "example-apicast"
+          },
+          "spec": {
+            "adminPortalCredentialsRef": {
+              "name": "mysecretname"
+            }
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/vmogilev_rhmi/apicast-operator:v0.8.0
```

Comment:
this is the correct image?

Reply:
@eguzki   Thank you! confusion. Release image is not created yet. I will try mark PR as draft